id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/exception/IdRepoExceptionHandler.java

```diff
@@ -105,7 +105,7 @@ protected ResponseEntity<Object> handleNoResourceFoundException(NoResourceFoundE
 				"handleNoResourceFoundException - \n" + ExceptionUtils.getStackTrace(Objects.isNull(rootCause) ? noResourceFoundException : rootCause));
 
 		IdRepoAppException idRepoAppException = new IdRepoAppException(INVALID_REQUEST.getErrorCode(), INVALID_REQUEST.getErrorMessage());
-			return new ResponseEntity<>(buildExceptionResponse(idRepoAppException, ((ServletWebRequest)request).getHttpMethod(), null), HttpStatus.OK);
+			return new ResponseEntity<>(buildExceptionResponse(idRepoAppException, ((ServletWebRequest)request).getHttpMethod(), null), HttpStatus.NOT_FOUND);
```

Comment:
Httpstatus should be 200 for all the api response.

Reply:
corrected